nss-day-cohort-60/rare-python-server-tyrannical-tators
# organize category list

This is part 1 of ticket 22. When client requests all categories, they will be ordered alphabetically by label.

Steps to Test
1. Fetch the [lh-category-server-list](https://github.com/nss-day-cohort-60/rare-python-server-tyrannical-tators/compare/main...lh-category-server-list?expand=1) branch and switch to it
2. Start server in debugger moder. 
3. Open postman and make request: http://localhost:8088/categories
4. Verify the response returns 6 dictionaries in alphabetical order by label
